pkg/cmd/cli/cmd/build.go

```diff
@@ -181,6 +181,6 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().BoolVarP(&(cfg.ForceCopy), "copy", "c", false, "Use local file system copy instead of git cloning the source url")
 	buildCmd.Flags().StringVar(&(cfg.RuntimeImage), "runtime-image", "", "Image that will be used as the base for the runtime image")
 	buildCmd.Flags().VarP(&(cfg.RuntimeArtifacts), "runtime-artifact", "a", "Specify a file or directory to be copied from the builder to the runtime image")
-
+	buildCmd.Flags().StringVarP(&(cfg.Network), "network", "", "", "Specify the default Docker Network name to be used in build process")
```

Comment:
> Actually, also I was afraid of breaking something that was working, so what I did was to create a new flag that could be used to exit the regular flow of creating a build container with the default network :)

yeah, i can understand why you did that :)  I'm just letting you know i think your fears are unwarranted.  The openshift build codepath doesn't go through the CLI config path.